pokemongo_bot/walkers/polyline_walker.py

```diff
@@ -1,17 +1,51 @@
 from __future__ import absolute_import
 from geographiclib.geodesic import Geodesic
+from random import uniform
+from time import sleep
 
 from pokemongo_bot.walkers.step_walker import StepWalker
 from .polyline_generator import PolylineObjectHandler
 from pokemongo_bot.human_behaviour import random_alt_delta
 
 
 class PolylineWalker(StepWalker):
+    def handle_flight(self):
+        speed = uniform(self.bot.config.fly_min, self.bot.config.fly_max)
+        origin_lat, origin_lng, origin_alt = self.bot.position
+        remaining = Geodesic.WGS84.Inverse(origin_lat, origin_lng, self.dest_lat, self.dest_lng)["s12"]
+        time_to_sleep = remaining / speed
+        sleep(time_to_sleep)
+        self.bot.api.set_position(self.dest_lat, self.dest_lng, self.dest_alt)
```

Comment:
There is only one class changing bot.position 

that is StepWalker.... it would be nice if we could keep it that way!!! in the end StepWalker is the super class...

Flight will teleport to destination ...
after sleeping the required time....

you can do that by using super(PolylineWalker, self).step(float("inf"))